CONTRIBUTING.md

```diff
@@ -1,35 +1,26 @@
+Contributions to this repository are intended to become part of the World Wide Web Consortium's (W3C) [HTML Recommendation](https://www.w3.org/TR/html52/), and are governed by the [W3C Patent Policy](http://www.w3.org/Consortium/Patent-Policy-20040205/) and [Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
+
+## Contributing
+
+Before making your first contribution, you should read the documentation on [editing conventions and the source documents](https://github.com/w3c/html/blob/master/docs/contributing-documentation.md). You may also want to read [how the spec is built](https://github.com/w3c/html/blob/master/docs/build-documentation.md).
+
+Editorial improvements are welcome, just make a [Pull Request (PR)](https://github.com/w3c/html/pulls)! You may encounter a warning about Intellectual Property Rights (IPR) commitments if you are not a member of the W3C [Web Platform Working Group (WebPlat WG)](https://www.w3.org/WebPlatform/WG/), but this will be handled by the WebPlat WG editorial team. Specific concerns about IPR and the W3C may be found in the [Intellectual Rights FAQ](https://www.w3.org/Consortium/Legal/IPR-FAQ-20000620.html).
+
+Your concern might already be known. Before beginning work, check [the issue tracker](https://github.com/w3c/html/issues) to see if there are any existing discussions about the issue. If you want to help but are unsure how to begin, some issues are marked as [good first issues](https://github.com/w3c/html/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) or [help wanted](https://github.com/w3c/html/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). These might be good places to start. 
+
+To make **substantive** contributions, changes that clarify or update conformance requirements, you must either participate in the W3C [Web Platform Working Group](https://www.w3.org/WebPlatform/WG/) or make a [non-member patent licensing commitment](https://www.w3.org/2004/01/pp-impl/83482/nmlc). 
+This is because the W3C Patent Policy depends on granting licenses for the [essential claims](https://www.w3.org/Consortium/Patent-Policy-20040205/#def-essential) needed to implement the spec. A list of [previous substantive contributions](https://github.com/w3c/html/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+label%3Asubstantive) can be found on the issue tracker.
+
+If your contribution includes the work of others, please see [Contributing Documentation](https://github.com/w3c/html/blob/master/docs/contributing-documentation.md for further guidance).
+
+## Tests
+
+For **normative changes**, a corresponding [web-platform-tests](https://github.com/w3c/web-platform-tests) PR is highly appreciated. The W3C defines a normative change as edits that affect the formal part of the Recommendation, rules that everyone must follow. **Non-normative** changes affect supplementary information and guidance fo these rules.
```

Comment:
normative and substantive is pretty similar. Technically, a normative change might not affect conformance because it uses a should/may level requirement, but I think we are better being consistent in terminology in this document.